### PR TITLE
HomePage: include information about the alumni benefits

### PR DESCRIPTION
**Pull request by annagav**
#### What are the relevant tickets?
Related to #4429. Modifications described in https://docs.google.com/document/d/1xYmIWqdISNZ74DNfS8ndLelHnsFQeYmBOlxBLP4Swfs/edit?usp=sharing
#### What's this PR do?
Modifies the home page to include information about the alumni benefits.

#### How should this be manually tested?
Go to the home page and check that the copy updates are correct and the links to "Benefits" are working.

REQUIREMENT: should have completed all the steps in PR #4432 to setup and publish a "Benefits" page.

### CODE REVIEW

**Comment by annagav** [comment]
Screen Shot 2019-10-23 at 2 31 21 PM

**Comment by annagav** [comment]
Screen Shot 2019-10-24 at 12 03 10 PM

**Comment by annagav** [comment]
@tasawernawaz I rebased this branch on master. It is ready for review now. Thanks!

**Comment by tasawernawaz** [comment]
@annagav I saw an extra `>` instead of closing anchor tag. I have pushed the fix, otherwise looks good. 👍 

**Updated screenshot.**
Screenshot 2019-10-28 at 6 20 44 PM


@pdpinch can I get you approve on the screenshot?

**Comment by annagav** [comment]
@tasawernawaz the deployment failed.

**Comment by tasawernawaz** [comment]
> @tasawernawaz the deployment failed.

I can take a look.

**Comment by tasawernawaz** [comment]
@annagav rebase did the trick.

**Comment by annagav** [comment]
@pdpinch, this PR was approved, but I am not sure if you had the chance to look at the screenshots. Let me know if I can merge this PR.

**Comment by pdpinch** [comment]
Thanks @annagav. I think we're going to have to do something about all that text... 

Also, will the link be conditional on the page existing? I have a feeling that we won't have the final text before we want to merge this.

File: cms/templates/cms/home_page.html [hunk 1]
```diff
@@ -125,8 +128,10 @@ <h4>Proctored Tests</h4>
             <div class="reasons-images-wrap">
               <img src="{% static 'images/hiw3.png' %}" alt="">
             </div>
-            <h4>Earn a Valuable Credential</h4>
-            <p>A MicroMasters credential from MITx proves that you’ve gained the knowledge and skills to give you a competitive advantage for career success.</p>
+            <h4>Earn a Valuable Credential and Boost Your Network</h4>
+            <p>A MicroMasters program credential from MITx proves that you’ve gained the knowledge
+              and skills and knowledge to stand out in your field and qualifies you as an affiliate of
+              the MIT Alumni Association.<br><a href="{{benefits_url}}">Learn More</a></p>
```
Comment (pdpinch) [inline cms/templates/cms/home_page.html]:
```suggestion
              the MIT Alumni Association. <a href="{{benefits_url}}">Learn More</a></p>
```

Remove the line break, so this section isn't _so_ much longer than the rest.

**Comment by pdpinch** [comment]
I spoke to @abdulkdawson -- besides my one code comment, can we also:
1. Move the titles (all 4) closer to the icons and
2. Reduce the line spacing between "Earn a Valuable Credential" and "and Boost your Network" to match.

File: cms/templates/cms/home_page.html [hunk 2]
```diff
@@ -87,9 +87,15 @@ <h3>What is the MicroMasters<span class="trademark">&reg;</span> Program Credent
...
-                apply for an accelerated, on campus, master’s degree program at MIT or other top universities.</p>
+                anywhere in the world. Learners who pass an integrated set of MIT graduate-level courses on edX.org,
+                and one or more proctored exams. This credential is valuable in and of itself.  Additionally,
+                credential-holders can then apply for an accelerated, on campus, master’s degree program at MIT
+                or other universities.
+                {% if benefits_url %}
+                  All MicroMasters program credential holders are also considered
+                  <a href="{{benefits_url}}">affiliates of the MIT Alumni Association</a>.
+                {% endif %}
+              </p>
```
Comment (tasawernawaz) [inline cms/templates/cms/home_page.html]: @pdpinch here I have added a condition to add last line only if `benefits_url` exists.

**Comment by tasawernawaz** [comment]
@pdpinch Added conditional rendering and CSS tweaks.

Screenshot 2019-10-29 at 3 26 21 PM

**Comment by pdpinch** [comment]
I'd like to get an opinion from @abdulkdawson about the CSS update: https://github.com/mitodl/micromasters/pull/4434#issuecomment-547356978

**Comment by pdpinch** [comment]
The icon and text layout still isn't great. Can we try it with the labels (e.g. "Online Learning" above the icons?

File: cms/templates/cms/home_page.html [hunk 3]
```diff
@@ -125,8 +131,12 @@ <h4>Proctored Tests</h4>
             <div class="reasons-images-wrap">
               <img src="{% static 'images/hiw3.png' %}" alt="">
             </div>
-            <h4>Earn a Valuable Credential</h4>
-            <p>A MicroMasters credential from MITx proves that you’ve gained the knowledge and skills to give you a competitive advantage for career success.</p>
+            <h4>Earn a Valuable Credential and Boost Your Network</h4>
+            <p>A MicroMasters program credential from MITx proves that you’ve gained the knowledge
+              and skills and knowledge to stand out in your field and qualifies you as an affiliate of
```
Comment (pdpinch) [inline cms/templates/cms/home_page.html]:
```suggestion
              and skills to stand out in your field and qualifies you as an affiliate of
```

Somehow we got "knowledge" twice

**Comment by tasawernawaz** [comment]
> The icon and text layout still isn't great. Can we try it with the labels (e.g. "Online Learning" above the icons?

@pdpinch I will try, but if I can get the design that would help instead of hit and trial.

**Comment by tasawernawaz** [comment]
@pdpinch this how it looks now,

Screenshot 2019-10-31 at 12 49 44 PM

**Comment by abdulkdawson** [comment]
@tasawernawaz can we put some space between the icons and the bottom text?

**Comment by tasawernawaz** [comment]
@abdulkdawson would 10px enough?

Screenshot 2019-10-31 at 6 37 22 PM

**Comment by abdulkdawson** [comment]
@tasawernawaz this latest version looks good to me.

**Comment by pdpinch** [comment]
The third icon and text looks lower than the rest. Can we at least align the text?

67951608-b13f4980-fc0d-11e9-889a-04ad0982a808

**Comment by tasawernawaz** [comment]
@pdpinch sorry my bad, fixed it.

**Pull request merged**